### PR TITLE
fix: Disable NavigationView compact mode, make pane wider

### LINKED ISSUE #135: [UWP/WASM][Menu] The ExpandingBottomSheet  and ModalstandardBottomsheet item is cut off  from Burger menu
## Current behavior
burger list cut off issue

Capture12

## Expected behavior
The item name must be fully displayed from the list

## How to reproduce it (as minimally and precisely as possible)

1. From the menu left select Components
2. Under Components section look for ExpandingBottomSheet item
3. Notice the item name is cut off

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [x] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [x] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): fixes https://github.com/unoplatform/uno/issues/5143, fixes https://github.com/unoplatform/Uno.Gallery/issues/131, fixes #135

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

CompactMode looks off.

## What is the new behavior?

Disabled the NavigationView compact mode by forcing the pane open and hiding the toggle button in expanded mode.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs [hunk 1]
```diff
@@ -1,149 +1,135 @@
...
-			}
-
-			if (NestedSampleFrame.CanGoBack)
-			{
-				NestedSampleFrame.GoBack();
-			}
-			else
-			{
-				NestedSampleFrame.Content = null;
+    public sealed partial class Shell : UserControl
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs:11]: might want to fix the space/tab settings, you just replaced all tabs by spaces

**Review by Xiaoy312 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs (hunk 1 above)
Comment (MartinZikmund) [reply Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs:11]: oh! Not sure why it does not apply project defaults

Comment (MartinZikmund) [reply Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs]: Fixed now

**Review by kazo0 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs (hunk 1 above)
Comment (Xiaoy312) [reply Uno.Gallery/Uno.Gallery.UWP/Views/Shell.xaml.cs]: because there is no `.editorconfig` in this project

**Pull request merged**